cmd/eksctl-anywhere/cmd/listovas.go

```diff
@@ -65,8 +65,17 @@ func listOvas(context context.Context, clusterSpecPath, bundlesOverride string)
 		return err
 	}
 
-	bundle := clusterSpec.VersionsBundle
+	for _, bundle := range clusterSpec.VersionsBundlesMap {
```

Comment:
this trusts that the versionsBundlesMap only has releases for the kubernetes versions in the cluster object. I think this is an internal implementation, it's probably a bad idea to make external callers rely on this. Can't we just get the unique kubernetes versions from the cluster and iterate over them?

Reply:
Or if you think it's something you use frequently enough, write a method in the spec like `ClusterVersionBundles` that returns such filtered collection